Kickstarter-iOS/Features/DashboardDeprecationBanner/DashboardDeprecationView.swift

```diff
@@ -0,0 +1,50 @@
+import Library
+import SwiftUI
+
+struct DashboardDeprecationView: View {
+  private let contentPadding = 12.0
+  private let imageSizeMultiplier = 1.5
+  private let deprecationDateText = formatted(dateString: "2023-08-14")
+
+  var body: some View {
+    HStack {
+      if let iconImage = image(named: "fix-icon", inBundle: Bundle.framework) {
+        Image(uiImage: iconImage)
+          .frame(width: contentPadding * imageSizeMultiplier)
+          .scaledToFit()
```

Comment:
This will ensure that the view is scaled to fit, so only need to set the width above. Also `resizable()` is no longer needed. SwiftUI modifiers work on the previous modifier's returned view. So `frame` --> `scaledToFit` will scaled a resized image. Before it was `resizable` -> `scaledToFit` -> `foregroundColor` -> `frame`. So could be reorganized to be a little cleaner.